src/cmds/stack/setup.js

```diff
@@ -53,13 +54,15 @@ const cpDevDir = (basePath, version) => {
 };
 
 const cloneRepositories = (initPath, repositories) => {
-  repositories.forEach((repo) => {
+  repositories.forEach((fullRepo) => {
+    const [repo, checkout] = fullRepo.split(':');
     const repoName = repo.split('/')[1];
     const repoPath = path.join(initPath, repoName);
     console.log(`Cloning: ${repoName}`);
     gitClone(
       `http://github.com/${repo}`,
       repoPath,
+      { checkout },
```

Comment:
Alligned with the previous comment, this should be udpated :point_down: 
```suggestion
      { checkout: branch },
```
>Im considering `{ checkout }` is a syntactic sugar for `{ checkout: checkout }`. If it isn't, ignore the suggestion but do correct the identified to reference `branch` instead of `checkout`.

Reply:
Done!